packages/trpc/server/routers/viewer/teams/inviteMember.handler.ts

```diff
@@ -41,7 +41,7 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     ? input.usernameOrEmail
     : [input.usernameOrEmail];
 
-  emailsToInvite.forEach(async (usernameOrEmail) => {
```

Comment:
we cannot throw error inside forEach so replaced it with for of loop

Reply:
sean already updated the invitmember.handler in his Billing PR so we can use that. not making any change here.